code/game/objects/effects/aliens.dm

```diff
@@ -127,28 +126,29 @@
 	anchored = TRUE
 	var/atom/acid_t
 	var/ticks = 0
-	var/acid_strength = 0.04 //base speed, normal
+	var/acid_strength = REGULAR_ACID_STRENGTH //how fast something will melt when subject to this acid.
 	var/acid_damage = 125 //acid damage on pick up, subject to armor
-	var/strength_t
+	var/strength_t = 4 //stages of meltage, currently used to determine what messages to output
+	var/acid_melt_multiplier //How much faster or slower acid melts specific objects/turfs.
```

Comment:
why no autodoc

Reply:
Why did you resolve this for my comment when you didn't add the autodoc 4head